source/codegen/metadata/nifgen/enums.py

```diff
@@ -101,41 +130,38 @@
     'ClockMode': {
         'values': [
             {
-                'documentation': {
-                    'description': 'High resolution sampling—Sample rate is generated by a high–resolution clock source.'
-                },
                 'name': 'NIFGEN_VAL_HIGH_RESOLUTION',
                 'value': 0
             },
             {
-                'documentation': {
-                    'description': 'Divide down sampling—Sample rates are generated by dividing the source frequency.'
-                },
                 'name': 'NIFGEN_VAL_DIVIDE_DOWN',
                 'value': 1
             },
             {
-                'documentation': {
-                    'description': 'Automatic Selection—NI-FGEN selects between the divide–down and high–resolution clocking modes.'
-                },
                 'name': 'NIFGEN_VAL_AUTOMATIC',
                 'value': 2
             }
         ]
     },
+	'CloseExtCal': {
```

Comment:
We dont need CloseExtCal values since we are not adding support for the API itself

Reply:
removed the enum